packages/tasit-action/src/contract/Action.js

```diff
@@ -180,16 +180,29 @@ export class Action extends Subscription {
       }
     };
 
-    let ethersListener;
-
-    if (once) {
-      ethersListener = async blockNumber => {
-        await baseEthersListener(blockNumber);
-        this.off(eventName);
-      };
-    } else {
-      ethersListener = baseEthersListener;
-    }
+    // Note:
+    // On the development env (using ganache-cli)
+    // Blocks are being mined simultaneously and generating a sort of unexpected behaviors like:
+    // - once listeners called many times
+    // - sequential blocks giving same confirmation to a transaction
+    // - false-positive reorg event emission
+    // - collaborating for tests non-determinism
+    //
+    // Tech debt:
+    // See if there is another way to avoid these problems, if not
+    // this solution should be improved with a state structure identifying state per event
+    //
+    // Question:
+    // Is possible that behavior (listener concurrency calls for the same event) be desirable?
```

Comment:
```suggestion
    // Is it possible that that behavior (listener concurrent calls for the same event) is desirable?
```

Reply:
@marcelomorgado This captures your intended meaning, right?

Reply:
Yep.

Reply:
Okay - can you please include this change in the next PR?

Reply:
Done on https://github.com/tasitlabs/TasitSDK/pull/383 (Now this comment lives on Subscription class)

Reply:
Thanks! Resolving this.